docs/docs.go

```diff
@@ -1,4 +1,4 @@
-// Package docs Code generated by swaggo/swag at 2024-02-08 16:02:27.560628 -0500 EST m=+1.587012584. DO NOT EDIT
```

Comment:
@JamesReate Are you okay with us removing the generated timestamp from our swagger docs to get consistent results for this new CI step?

Reply:
yeah that's good!